application/modules/home/controllers/Home.php

```diff
@@ -43,4 +43,12 @@ public function index() {
 		// load view
 		$this->load_view('top', $this->data);
 	}
+
+	public function side_ci()
+	{
+		$this->data["exhibition"]["name"] = "test";
+
+		 // load view
+		$this->load_view('side_ci', $this->data);
+	}
```

Comment:
__[code_sniffer]__
* No comment block marks the end of file instead of the closing PHP tag. Please add a comment block containing only "End of file Home.php".
* No comment block marks the end of file instead of the closing PHP tag. Please add a comment block containing only "Location: ./modules/home/controllers/Home.php".

<!-- Comment by [SideCI](https://sideci.com) -->